lib/ui/views/settings_view.dart

```diff
@@ -44,130 +44,340 @@ class _SettingsPageState extends State<SettingsPage> {
-                  thickness: 2.0,
-                ),
-                Column(
-                  children: <Widget>[
-                    Text(
-                      "App details",
-                      style: Theme.of(context).textTheme.headline5,
-                    ),
-                    Text(
-                      "v0.1.3",
-                      style: Theme.of(context).textTheme.headline6,
-                    ),
-                    Divider(
-                      thickness: 1.0,
-                    ),
-                    RichText(
-                      text: TextSpan(
-                        children: [
-                          TextSpan(
-                            text: 'Please submit issues into the ',
-                            style: Theme.of(context).textTheme.bodyText1,
-                          ),
-                          TextSpan(
-                            text: 'Github issues page.',
-                            style:
-                                TextStyle(color: Colors.blue, fontSize: 16.0),
-                            recognizer: TapGestureRecognizer()
-                              ..onTap = () {
-                                launch(
-                                    'https://github.com/voteflux/voting_app/issues/new/choose');
-                              },
-                          ),
-                        ],
-                      ),
-                    ),
-                    RichText(
-                      text: TextSpan(
-                        children: [
-                          TextSpan(
-                            text: 'Join us on ',
-                            style: Theme.of(context).textTheme.bodyText1,
-                          ),
-                          TextSpan(
-                            text: 'Discord',
-                            style:
-                                TextStyle(color: Colors.blue, fontSize: 16.0),
-                            recognizer: TapGestureRecognizer()
-                              ..onTap = () {
-                                launch('https://discord.gg/xFMUTZw');
-                              },
-                          ),
-                        ],
-                      ),
-                    ),
-                  ],
-                )
-              ],
+                  )
+                ],
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _account() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(
+          height: 30,
+          child: ListTile(
+            contentPadding: EdgeInsets.zero,
+            title: Text(
+              "Account",
+              style: TextStyle(
+                  color: Theme.of(context).colorScheme.primary,
+                  fontSize: 14,
+                  fontWeight: FontWeight.w700),
+            ),
+          ),
+        ),
+        SizedBox(
+          height: 30,
```

Comment:
This SizedBox pattern doesn't really work - The click event isn't on top of the actual arrow, as the sized box is restricting the height of the list tile. Can you look for a different solution. Either remove the height or use a different widget.

Reply:
No problem, I will see how to solve it tonight

Reply:
Hi mate, could you please check the modification and see if that is ok now?